man/systemd.netdev.xml

```diff
@@ -344,7 +344,8 @@
         <varlistentry>
           <term><varname>DefaultPVID=</varname></term>
           <listitem>
-            <para>This specifies the default port VLAN ID of a newly attached bridge port.</para>
+            <para>This specifies the default port VLAN ID of a newly attached bridge port.
+            Set this to <literal>none</literal> or <literal>0</literal> to disable the PVID.</para>
```

Comment:
I don't see the point of allowing both. IIUC, "none" is translated to 0, so "0" should not be allowed as an option here.

`vlanid_is_valid` allows 0 and has a comment about explicitly allowing 0. Looks like this should be changed.